content/tutorial-ma.md

```diff
@@ -264,25 +264,26 @@ Now, if we want to create a very simple approximation for this data, we should t
 dates[~china_total.mask]
 ```
 
-Finally, we can use the [numpy.polyfit](https://numpy.org/devdocs/reference/generated/numpy.polyfit.html#numpy.polyfit) and [numpy.polyval](https://numpy.org/devdocs/reference/generated/numpy.polyval.html#numpy.polyval) functions to create a cubic polynomial that fits the data as best as possible:
+Finally, we can use the
+[fitting functionality of the numpy.polynomial](https://numpy.org/doc/stable/reference/generated/numpy.polynomial.polynomial.Polynomial.fit.html)
```

Comment:
Could we use intersphinx for links like this? (not in scope in this PR, but this is where I spotted it)

Reply:
We could, but we've chosen not to - the reason being that the MyST extended features are currently tied to sphinx, meaning they only work for the outputs that pass through the sphinx build process. In practice this means that it would render correctly in the website, but *not* in the Jupyter notebook (i.e. for users running tutorials locally or on binder).

As of now we've decided to limit the tutorials only to syntax that will render correctly in both the browser and the html, but we could certainly revisit this at some point.

PS - this is also one of the major reasons I'm excited about [MystJS](https://www.myst.tools/docs/mystjs), though I'm not sure that support for sphinx extensions like intersphinx are on the roadmap (yet).

Reply:
Oh, gotcha, my ignorance shows how much I dislike working in the browser with notebooks :)

Reply:
There are certainly things we could consider that might work within the current constraints. E.g. trying thebe instead of binder for cell-level interactivity rather than notebook-level might work, and would have the advantage that the sphinx-specific functionality (admonitions, links, etc.) could be used more extensively, as readers would be working in the website directly. I don't have any real idea because I've never tried!

Reply:
No worries about brute forcing a workaround for this. 

Adding a link checker CI job would address my main concern about the hard-wired URLs, so I think we should just do that instead (I think we should do it anyway).